src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java

```diff
@@ -0,0 +1,14 @@
+package com.wcc.platform.domain.cms.pages;
+
+import com.wcc.platform.domain.cms.attributes.Contact;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+// TODO: Get photo for About Us hero section.
```

Comment:
Take from Figma, you can upload to Google drive and we can use the link for download, similar to the landing page test.